code/cgame/cg_playerstate.c

```diff
@@ -463,7 +468,9 @@ void CG_CheckLocalSounds( playerState_t *ps, playerState_t *ops ) {
 	}
 
 	// fraglimit warnings
-	if (!CG_IsATeamGametype(cgs.gametype)) {
+	if (!CG_UsesTeamFlags(cgs.gametype) && !CG_UsesTheWhiteFlag(cgs.gametype) &&
+			!CG_IsARoundBasedGametype(cgs.gametype) && cgs.gametype != GT_DOUBLE_D &&
+			cgs.gametype != GT_DOMINATION) {
```

Comment:
Might want to add a CG_GametypeUsesFraglimit and limit it to GT_FFA, GT_TOURNEY, GT_TEAM and GT_LMS.

This new line is very hard to read.

Reply:
I did something like that past week!
However I haven't opened up a PR because I haven't uploaded the branch to my fork yet. I still have to iron out some things before uploading.

Reply:
For now you could just simplify to simply checking for GT_FFA, GT_TOURNEY, GT_TEAM and GT_LMS. 4 simple checks instead of 5 advanced ones.

Reply:
In all reality I wasn't sure if I should submit that until you mentioned the idea. I'll rough around the edges and submit the results, with a plus.

Reply:
#107 is my proposal for this particular change.